examples/src/index.tsx

```diff
@@ -111,6 +114,35 @@ class App extends React.Component<{}, State> {
               Flip Vertical
             </label>
           </div>
+          <button
+            id="horizontal-center-button"
+            onClick={() => {
+              this.setState({
+                crop: { ...this.state.crop, x: 0 },
+              })
+            }}
+          >
+            Center Horizontally
+          </button>
+          <div>
+            <p>Crop Area:</p>
+            <div>
+              {['x', 'y', 'width', 'height'].map((attribute: string) => {
+                if (!this.state.croppedArea) {
+                  return null
+                }
+
+                return (
+                  <div key={attribute}>
+                    {attribute}:
+                    <b id={`crop-area-${attribute}`}>
+                      {Math.round(this.state.croppedArea[attribute])}
+                    </b>
+                  </div>
```

Comment:
@ValentinH I've also added this little HTML by the top of the example:

![Screenshot from 2020-10-31 23-52-46](https://user-images.githubusercontent.com/5278570/97794108-4a63c000-1bd4-11eb-9307-7b077a15bd9c.png)

This made Cypress testing simpler, as I can actually debug and access the real output from `onCropComplete` and `onCropAreaChange` functions. Does it make sense?